qutip/floquet.py

```diff
@@ -612,7 +612,8 @@ def floquet_master_equation_rates(f_modes_0, f_energies, c_op, H, T,
     Gamma = np.zeros((N, N, M))
     A = np.zeros((N, N))
 
-    nT = 100
+    # time steps for integration of coupling operator
+    nT = int(np.max([2 * kmax * np.ceil(omega), 100]))
```

Comment:
This formula for `nT` looks odd to me. If I plug it into `dT = T / nT` I get `dT ~ T**2`.

`omega` shouldn't affect `nT` because `omega * t` always traces out one period over the interval of the integral, i.e. `t = [0, T]`. So the full number of periods of `e**(1j * k * omega * t)` is just `k`.

This seems to suggest that we should use `nT = max(20 * kmax, 100)` or similar to give a fixed number of steps per period.

Reply:
Thanks for the hint. I did not see that the number of periods of the exponential is fixed by `k` only.  I'll change the definition of `nT`